### PR TITLE
Adding Kotlin Tutorial 3 text.

### PR DESCRIPTION
**Pull request by SHeimlich**

### CODE REVIEW

File: particles/Tutorial/Kotlin/README.md [hunk 1]
```diff
@@ -135,3 +135,86 @@ arcs_kt_binary(
...
+
+
+// Unlike previous recipes, this one includes two particles.
+recipe RenderSlotsRecipe
+  ParentParticle
+    // The ParentParticle consumes root just like particles in previous examples.
+    consume root
+      // ParentParticle also provides mySlot. Note the additional tab over.
+      // The name "slot" is the name the recipe uses for the slot, and is how
+     // the recipe connects the parent's mySlot and the child's mySlot
```
Comment (csilvestrini) [inline particles/Tutorial/Kotlin/README.md]: fix indenting

File: particles/Tutorial/Kotlin/3_RenderSlots/ParentParticle.kt [hunk 2]
```diff
@@ -8,9 +8,7 @@ import kotlin.native.internal.ExportForCppRuntime
  * Sample WASM Particle.
  */
 class ParentParticle : Particle() {
-    override fun getTemplate(slotName: String): String {
-        return """<b>Hello:</b><div slotId="mySlot"></div>"""
-    }
+    override fun getTemplate(slotName: String) = """<b>Hello:</b><div slotId="mySlot"></div>"""
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/3_RenderSlots/ParentParticle.kt]: nit: We don't need `"""` here.

File: particles/Tutorial/Kotlin/README.md [hunk 3]
```diff
@@ -135,3 +135,86 @@ arcs_kt_binary(
...
+    // And the ChildParticle consumes the slot.
+    consume mySlot as slot
+
+  description `Javascript Tutorial 3: Render Slots`
+```
+
+Next, we implement the parent and child particles in Kotlin. To do this, we will create a Kotlin file for each particle as outlined in the Arcs manifest file. We start with the ParentParticle which provides the slot.
+```Kotlin
+class ParentParticle : Particle() {
+    // The parent particle needs to provide a div with slotid "mySlot". This is where the child particle will be rendered.
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/README.md]: nit: You would not write such a long comment in code

File: particles/Tutorial/Kotlin/README.md [hunk 4]
```diff
@@ -135,3 +135,86 @@ arcs_kt_binary(
...
+    override fun getTemplate(slotName: String) = """<b>Hello:</b><div slotId="mySlot"></div>"""
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/README.md]: `"""` is unnecessary

File: particles/Tutorial/Kotlin/README.md [hunk 5]
```diff
@@ -135,3 +135,86 @@ arcs_kt_binary(
...
+}
+
+@Retain
+@ExportForCppRuntime("_newParentParticle")
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/README.md]: no need to specify the name here again, just `@ExportForCppRuntime`

File: particles/Tutorial/Kotlin/README.md [hunk 6]
```diff
@@ -135,3 +135,86 @@ arcs_kt_binary(
...
+```
+
+The ChildParticle looks nearly identical to the particles we created in our first tutorials.
+```Kotlin
+class ChildParticle : Particle() {
+    override fun getTemplate(slotName: String) = "Child"
+}
+
+@Retain
+@ExportForCppRuntime("_newChildParticle")
```
Comment (piotrswigon) [inline particles/Tutorial/Kotlin/README.md]: ditto

**Pull request merged**